sdk/python/responsible-ai/vision/responsibleaidashboard-object-detection-MSCOCO.ipynb

```diff
@@ -64,7 +64,7 @@
     "    urlretrieve(data_url, filename=data_output_path)\n",
     "\n",
     "\n",
-    "annotations = \"msCOCOValExample.jsonl\"\n",
+    "annotations = \"msCOCOValExample7.jsonl\"\n",
```

Comment:
is this change on purpose?

Reply:
yes because the mscoco dataset kept failing in the notebook due to new error checking logic (explanation was not getting computed due to num_masks being too low) - if I bumped up num_masks too high thought the notebook will take way too long.  Hence, I reduced number of images but still bumped up num_masks.